snapshot/cli/cli.go

```diff
@@ -119,3 +127,36 @@ func (c *ingestGitCommitCommand) run(db snapshot.DB, _ *cobra.Command, _ []strin
 	fmt.Println(id)
 	return nil
 }
+
+type exportGitCommitCommand struct {
+	id string
+}
+
+func (c *exportGitCommitCommand) register() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "export_git_commit",
```

Comment:
Oh, I see, yeah, you're right...

I hope that eventually scoot-snapshot-db will just be transparently talking to a daemon, so I think the semantics here are reasonable, if awkward for now.

WDYT?